duckduckgo/iOS
# Fix assertion failure for missing UA

Task/Issue URL: https://app.asana.com/0/414709148257752/1204258831104817/f

**Description**:
- fix assertion failure
- make APIRequest.Headers API more strict

**Steps to test this PR**:
1. Put a breakpoint in any request and see if the UA is being sent correctly.
2. Check if autofill request (the one causing assertion) is getting a proper response.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
